website/content/ja/wgs/platforms/whitepaper/index.md

```diff
@@ -6,496 +6,261 @@ description: "この論文は、エンタープライズのリーダー、エン
-
-Platforms are particularly relevant for cloud-native architectures because they
-separate supporting capabilities from application-specific logic more than
-previous paradigms. In cloud-like environments resources and capabilities are
-often managed independently and integrated with custom business components; such
-resources may include databases and object stores, message queues and brokers,
-observability collectors and dashboards, user directories and authentication
-systems, task runners and reconcilers and more. An internal platform provides
-these to enterprise teams in ways that make them easy to integrate in their
-applications and systems.
-
-### Platform maturity
-
-At their most basic, internal platforms provide consistent experiences for
-acquiring and using individual services such as a pipeline runner, a database
-system or a secret store. As they mature internal platforms also offer
-_compositions_ of such services as self-serviceable templates for key scenarios
-like web application development or data analysis, aka MLOps.
-
-Use cases an enterprise could meet with platforms might progress through the
-following:
-
-1. Product developers can provision capabilities on demand and immediately use
-   them to run systems, such as compute, storage, databases or identities.
-1. Product developers can provision service spaces on demand and use them to run
-   pipelines and tasks, to store artifacts and configuration, and/or to collect
-   telemetry.
-1. Administrators of third-party software can provision required dependencies
-   like databases on demand and easily install and run that software.
-1. Product developers can provision complete environments from templates
-   combining run-time and development-time services required for specific
-   scenarios, such as web development or MLOps.
-1. Product developers and managers can observe functionality, performance, and
-   cost of deployed services through automatic instrumentation and standard
-   dashboards.
-
-By offering consistent, compliant experiences for individual capabilities or
-sets of them, internal platforms ultimately make it easier and more efficient
-for their users to deliver valuable products.
+1. なぜプラットフォームなのか?
+1. プラットフォームとはなにか？
+1. 成功するプラットフォームの特徴
+1. 成功するプラットフォーム・チームの特徴
+1. プラットフォームを導入する時の課題
+1. プラットフォームの成功をどう測定するか
+1. プラットフォームの能力
+
+## なぜプラットフォームなのか?
+
+今日のクラウドコンピューティングの世界において、プラットフォームとプラットフォームエンジニアリングは人気のある話題です。
+プラットフォーム構築の定義、テクニック、測定方法に飛び込む前に、まずプラットフォームがもたらす価値を追求することが重要です。
+
+過去20～30年にわたるプロセスの改善により、コンピュートやネットワーク、ストレージのようなインフラストラクチャーと、ビルドやテスト、デリバリー、オブザーバビリティのような開発者サービスの両方に、柔軟なサービスを提供するようになり、ソフトウェアアプリケーションやプロダクトチームのアジリティは大幅に向上しました。
+
+また、このような自主性とプロセス改善は、サービスをサポートする責任の多くを徐々にプロダクトチームに移し、彼らがインフラストラクチャーの課題に費やす時間と認知エネルギーを増やすことを余儀なくされ、組織にとって価値のある成果を生み出すための時間が減らすという効果をもたらしました。
+
+デリバリーチームを本来のコア業務に集中させ、組織全体の重複作業を減らしたいという要望から、企業をクラウドネイティブコンピューティング向けのプラットフォームを導入するようになりました。プラットフォームに投資することで、企業は次のことが可能になります：
+
+1. プロダクトチームの認知負荷を軽減し、製品開発とデリバリーを加速します。
```

Comment:
Indeed. I thought it would be better to unify one or the other, so "product team/developers/teams/owner" became "プロダクトXX". But the word "product" occurs in many places. Should we "puroduct" to be "プロダクト" rather than "製品"?

Reply:
IMO it should be translated into a single word so in this case "プロダクト"

Reply:
Understood. If there is consistency throughout the text we should leave it as is.